modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -32,15 +40,25 @@ CommerceOrganizationDisplayContext commerceOrganizationDisplayContext = (Commerc
 	<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
 	<aui:input name="redirect" type="hidden" value="<%= configurationRenderURL %>" />
 
-	<%
-	request.setAttribute("configuration.jsp-configurationRenderURL", configurationRenderURL);
-	request.setAttribute("configuration.jsp-redirect", redirect);
-	%>
-
 	<liferay-frontend:edit-form-body>
-		<aui:fieldset markupView="lexicon">
-			<aui:input label="root-organization-id" name="preferences--rootOrganizationId--" type="text" value="<%= commerceOrganizationDisplayContext.getRootOrganizationId() %>" />
-		</aui:fieldset>
+		<div class="form-group">
+			<label><liferay-ui:message key="root-organization" /></label>
+
+			<div id="<%= wrapperId %>"></div>
+		</div>
+
+		<liferay-frontend:component
```

Comment:
Hey, @FabioDiegoMastrorilli, any particular reason to use `liferay-frontend:component` instead of `react:component` directly?

This should free you from having to `render` your component manually and makes it easier to consistently provide a common basis for react components.

Reply:
It should even help you get rid of that `wrapperId` param and unnecessary placeholder `div`, unless you're doing something else with it!

Reply:
Yes! You're totally right :) In the first implementation I thought I needed to pass some functions as params to the component, so I used `liferay-frontend:component`, then I left it there. In the next PR it'll be replaced as you suggested.

Thank you (and welcome back)!

Reply:
@jbalsas I just realised I already tried it but it didn't work. 

In this case I'm calling a js component who's located in another module. It isn't possible to be done with `react:component` since it requires the component to be in the same module. 

I've asked [here](https://liferay.slack.com/archives/CNBG06JS3/p1627381090412700)

Reply:
> In this case I'm calling a js component who's located in another module. It isn't possible to be done with react:component since it requires the component to be in the same module.

In that conversation you rightly concluded that a bridge was needed and that's what you built here. I think you should be able to safely use `react:component`.

Reply:
Gotcha, so I still need a js file in the module where I import the module and I export it as it is :O 

```js
import Component from 'other/module';

export default Component
```

It totally makes sense now. Thank you : D